.env.example

```diff
@@ -0,0 +1,3 @@
+GOOGLE_OAUTH_URL=http://api.dev.commonground.tw/api/oauth2/google
```

Comment:
This environment variable will not be visible to the frontend if you don't explicitly add the prefix `NEXT_PUBLIC_`.